### DISCUSSION TITLE
Running mavlink-router on a Windows PC?

### DISCUSSION THREAD

**Discussion by pkvaloy** [mavlink-router/mavlink-router#379] [Q&A]
Hi. I'm a long time Windows user. Today we use MavProxy (for merging telemetry links) and Mission Planner at our groundstations(s).
I'm not entirely happy using MavProxy for this purpose. Is there a way for a non Linux expert to run mavlink-router in the linux subsystem in windows? What linux variants would I have to install to make this work? Thanks in advance!

**Comment by jbeyerstedt** [comment]
Basically everything you need to know to build and run mavlink-router should be in the readme. I haven't tried it myself in WSL, but it should work just fine. I just don't know how serial connections are handled in it, it you need that.

Regarding the linux distribution: That's up to you unfortunately. The readme shows which how to install the build tools for Debian, Fedora and ArchLinux (see https://github.com/mavlink-router/mavlink-router#fetch-dependencies). I would personally choose the latest Debian for any command-line based usage. Some packages might be a bit old, but I'm used to the way it works. But Fedora increased in popularity as a desktop distribution recently and is probably a good choice, too. ArchLinux is bleeding edge all the way, so I wouldn't recommend it to you.
